libr/arch/p/arch_riscv.c

```diff
@@ -75,7 +73,7 @@ static void get_riscv_args(riscv_args_t *args, const char *d, insn_t l, ut64 pc)
 			d++;
 			switch (*d) {
 			case 'd':
-				snprintf (RISCVARGN (args), RISCVARGSIZE , "%d", rd);
+				RISCVPRINTF ("%d", rd);
```

Comment:
replace this in all the other snprintf calls

Reply:
Fixed!